source-jira/README.md

```diff
@@ -0,0 +1,91 @@
+# Jira source connector
```

Comment:
We need to update the github workflow so that tests run in CI and the connector image gets built.

See [this commit](https://github.com/estuary/connectors/commit/8e69ce33fa8671087d7ad6d6ecae5cbcc94f4f88) for a reference.